extension/manifest.json

```diff
@@ -6,7 +6,9 @@
   "author": "Federico Brigante",
   "default_locale": "en",
   "permissions": [
-    "storage"
+    "storage",
+    "cookies",
+    "https://github.com/"
```

Comment:
The permission `"https://github.com/"` is implied by the `content_scripts`, it shouldn't be necessary.